frontend/src/app/pages/category-create/category-create.component.ts

```diff
@@ -45,11 +47,15 @@ export class CategoryCreateComponent implements OnInit {
     dialogRef.afterClosed().subscribe((res) => {
       switch (res.event) {
         case "confirm-option":
-          this.categoryService.createCategory(this.form.getRawValue()).subscribe(() => {
-            JSON.stringify(this.form.value);
-            this.router.navigate(['admin/categories']);
-          });
+          this.categoryService.createCategory(this.form.getRawValue()).pipe(
+            takeUntil(this.unsubscribe)
+          )
+            .subscribe(() => {
+              JSON.stringify(this.form.value);
```

Comment:
result of `JSON.stringify(this.form.value);` is ignored. You have this line in 4 places total